pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -216,8 +216,16 @@ func (r *{{ .Resource.Kind }}Reconciler) deploymentFor{{ .Resource.Kind }}({{ lo
 
 // labelsFor{{ .Resource.Kind }} returns the labels for selecting the resources
 // belonging to the given  {{ .Resource.Kind }} CR name.
+// Note that the labels follows the standards defined in: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
 func labelsFor{{ .Resource.Kind }}(name string) map[string]string {
-	return map[string]string{"type": "{{ lower .Resource.Kind }}", "{{ lower .Resource.Kind }}_cr": name}
+	return map[string]string{"app.kubernetes.io/name": "{{ lower .Resource.Kind }}",
+		"app.kubernetes.io/instance": name,
+		"app.kubernetes.io/version": "",
+		"app.kubernetes.io/component": "{{ .Resource.Kind }}",
+		"app.kubernetes.io/part-of": "",
```

Comment:
```suggestion
		"app.kubernetes.io/part-of": "", // here we need to add the Project name
```

Reply:
On top of the template add `machinery.ProjectNameMixin`

Then we can get the project name here `{{ .ProjectName }}`